src/client/containers/SettingsModal.tsx

```diff
@@ -77,6 +78,9 @@ export const SettingsModal: React.FC = () => {
   const updateNotesSortStrategyHandler = (selectedOption: any) => {
     _updateNotesSortStrategy(selectedOption.value)
   }
+  const updateNotesDirectionHandler = (selectedOption: any) => {
```

Comment:
This should apply to the preview was well.

Reply:
Unfortunately the "react-markdown" doesn't have options for set the text direction. But i use the direction rule of CSS.

Reply:
Great! Thanks.